### DISCUSSION TITLE
Double Click on Tag to Focus Tags Pane

### DISCUSSION THREAD

**Discussion by utkonos** [Vector35/binaryninja-api#5135] [Ideas]
If I have the Tags pane open and I double click on a tag in the linear or graph view, nothing happens. It would feel natural for this double click to focus on that particular tag in the tag pane. This is especially useful when leaning heavily on tags.

```python
>>> len(bv.tags)
425
```

**Comment by psifertex** [comment]
Is the tags pane popped out or is it docked?

We recently fixed a regression for that if the window wasn't docked. It will be in the new stable out soon.

**Reply by psifertex** [reply]
Oh wait, I misunderstood. You're not clicking the tag in the pane but clicking the tab in the view. Gotcha.

**Reply by utkonos** [reply]
Yes, clicking on the tag in the view.

**Comment by plafosse** [comment]
Created an issue from this discussion. https://github.com/Vector35/binaryninja-api/issues/5175
